WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -970,7 +963,7 @@ protected Address doInBackground(Double... args) {
         }
 
         protected void onPostExecute(@Nullable Address address) {
-            if (!isAdded() || address == null || address.getMaxAddressLineIndex() == 0) {
+            if (address == null || address.getMaxAddressLineIndex() == 0) {
```

Comment:
I was actually wondering if this was the case since we used to rely on `getActivity()` to pull resources. Thanks so much for the clarification!